02-express-tutorial/controllers/people.js

```diff
@@ -0,0 +1,61 @@
+let { people } = require('../data');
+
+const getPeople = (req, res) => {
+  res.status(200).json({ success: true, data: people });
+};
+
+const createPerson = (req, res) => {
+  const { name } = req.body;
+  if (!name) {
+    return res
+      .status(400)
+      .json({ success: false, msg: 'please provide a name' });
+  }
+  res.status(201).json({ success: true, person: name });
+};
+const createPersonPostman = (req, res) => {
+  const { name } = req.body;
+  if (!name) {
+    return res
+      .status(400)
+      .json({ success: false, msg: 'please provide a name' });
+  }
+  res.status(201).json({ success: true, data: [...people, name] });
+};
+
+const updatePerson = (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
+  const person = people.find((person) => person.id === Number(id));
+  if (!person) {
+    return res
+      .status(404)
+      .json({ success: false, msg: `person not found with this ${id}` });
+  }
+  const newPeople = people.map((person) => {
+    if (person.id === Number(id)) {
```

Comment:
Here's another place to fix a double-call to Number(). Also, is there a different function we can use instead?

Reply:
Not that we need to use a different fn, but are you aware of a different fn to turn a string into a number?